srcpkgs/rust-analyzer/template

```diff
@@ -11,11 +11,11 @@ maintainer="Gabriel Sanches <gabriel@gsr.dev>"
 license="Apache-2.0, MIT"
 homepage="https://rust-analyzer.github.io/"
 distfiles="https://github.com/${pkgname}/${pkgname}/archive/${_ver}.tar.gz"
```

Comment:
```suggestion
distfiles="https://github.com/rust-lang/rust-analyzer/archive/${_ver}.tar.gz"
```
rust-analyzer is part of the rust-lang namespace now

Reply:
Thanks!